### PR TITLE
Small changes in SubmitSLURM 

### PR DESCRIPTION
**Pull request by javierggt**
These are relatively trivial changes, I believe. It requires that one decides whether to create a directory or throw an exception though.

### CODE REVIEW

File: submit.py [hunk 1]
```diff
@@ -357,6 +359,8 @@ def write_general_header(self, f, mem=3000, walltime_hours=14, disk=1,
             self.write_option(f, "--partition=%s" % self.config['Cluster']["partition"])
         self.write_option(f, "--time=%d:00:00" % walltime_hours)
         if self.config["Mode"]["debug"]:
+            if not os.path.isdir("%s/out/"%os.getcwd()):
+                os.mkdir("%s/out/"%os.getcwd())
```
Comment (dsschult) [inline submit.py]: Prefer `os.path.join(os.getcwd(), 'out')`

Comment (javierggt) [inline submit.py]: I agree. The same applies all over the place.

Comment (briedel) [inline submit.py]: Someone volunteering? :D

Comment (javierggt) [inline submit.py]: and while one is changing % for os.path.join, what about changing % by format? :-)

Comment (briedel) [inline submit.py]: Format is a little crippled in 2.6. If we had 2.7 everywhere I would change it

**Pull request closed without merging**